zproject/wsgi.py

```diff
@@ -17,11 +17,14 @@
 from os.path import dirname, abspath
 import sys
 
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "zproject.settings")
+import django
+django.setup()
+
```

Comment:
Why is this being moved up?  I'm pretty sure that `django` won't import before the `import scripts.lib.setup_path_on_import` line.